app/views/clients/va/_tab_navigation.haml

```diff
@@ -0,0 +1,40 @@
+- unless ajax_modal_request?
+  :ruby
+    show_health_pilot = @client.show_health_pilot_for?(current_user) rescue false
+    show_health_hpc = @client.show_health_hpc_for?(current_user) rescue false
+    show_demographic = @client.show_demographics_to?(current_user) rescue false 
+    show_expanded_va_dashboard = @client.release_current_status == @client.class.full_consent_string
```

Comment:
Since were already in the context of VA here, let's just refer to this as 
```suggestion
    show_expanded_dashboard = @client.release_current_status == @client.class.full_consent_string
```